examples/platforms/nrf52811/README.md

```diff
@@ -0,0 +1,221 @@
+# OpenThread on nRF52811 Example
+
+This directory contains platform drivers for the Nordic Semiconductor nRF52811 SoC.
+This SoC is ment to be used in the Host / IEEE 802.15.4 Controller configuration in which the full OpenThread stack
+is running on the Host Processor and nRF52811 acts as a IEEE 802.15.4 radio running minimal OpenThread
+implementation allowing for communication between Host Processor and nRF52811.
+This firmware variant will be refered as RCP (Radio Co-Processor) or NCP-Radio.
+
+This platform is currently under development.
+
+## Toolchain
+
+Download and install the [GNU toolchain for ARM Cortex-M][gnu-toolchain].
+
+[gnu-toolchain]: https://launchpad.net/gcc-arm-embedded
+
+To install the GNU toolchain and its dependencies,
+run the following commands in Bash:
+
+```bash
+$ cd <path-to-openthread>
+$ ./script/bootstrap
+```
+
+## Building the examples
+
+Examples consist of two parts: firmware that is flashed to the nRF52811 SoC
+and host executables that are supposed to be executed on a POSIX platform in case of RCP usage.
+
+### Building the firmware
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+$ make -f examples/Makefile-nrf52811
+```
+
+After a successful build, the `elf` files can be found in
+`<path-to-openthread>/output/nrf52811/bin`.  
+You can convert them to hex using arm-none-eabi-objcopy`:
+```bash
+$ arm-none-eabi-objcopy -O ihex ot-cli-mtd ot-cli-mtd.hex
+$ arm-none-eabi-objcopy -O ihex ot-ncp-radio ot-ncp-radio.hex
+```
+
+### Building host executables
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+$ make -f src/posix/Makefile-posix
+```
+
+After a successful build, executables can be found in
+`<path-to-openthread>/openthread/output/posix/<system-architecture>/bin`.
+It is recommended to copy them to `/usr/bin` for easier access.
+
+## Building the examples with native SPI Slave support
+
+You can build the libraries with support for the native SPI Slave.
```

Comment:
The point is that the build needs to be made with the option enabled. Reading this I have an impression that NCP_SPI can be enabled later, but the fact is that it can not. Keeping `build` and `option enabled` in one sentence is crucial here.